### PR TITLE
Fix: Move buildkite pipeline from AWS to GCP

### PR DESCRIPTION
**Pull request by andrei-near**
Moving public nearcore pipeline to GCP buildkite agents.

### CODE REVIEW

**Comment by anshal-savla** [comment]
@andrei-near Can you please add some context as to why we are doing this?

Also, how are the GCP buildkite agents managed?

**Comment by andrei-near** [comment]
> 

This is to consolidate our efforts in moving away from AWS to GCP but also provide an isolated environment for public buildkite pipelines.

Currently the VMs are ran as part of a manually defined auto-scalling Instance Group (min 1, max 10) in US central.
Planning to move these resources later to terraform.

**Review by anshal-savla — APPROVED** [review]

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -11,19 +11,17 @@ steps:
       git fetch origin
       buf breaking --against ".git#ref=`git merge-base origin/master HEAD`"
 
-    timeout: 30
     agents:
-    - "distro=amazonlinux"
+      - "queue=gcp-standart"
```
Comment (wacban) [inline .buildkite/pipeline.yml:15]: Call my crazy but shouldn't it be spelled standar**d**?

**Review by wacban — APPROVED** [review]

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (andrei-near) [reply .buildkite/pipeline.yml:15]: you're sure not. I used standarD everywhere else except queue names, will change it as well

**Pull request merged**